docs/kubernetes/windows_demo.md

```diff
@@ -0,0 +1,227 @@
+    ```bash
+    git checkout master
+    git remote add demo https://github.com/almos98/gcp-filestore-csi-driver
+    ```
+2. Apply changes to personal fork.
+    ```bash
+    git fetch demo refs/heads/windows_demo
+    git checkout -b csi_prototype
+    COMMIT=$(git ls-remote demo | grep refs/heads/windows_demo | cut -f 1)
+    git cherry-pick $COMMIT
+    ```
+3. Build Windows binary and upload it to the bucket.
+    ```bash
+    make windows-local
+    gsutil cp bin/gcfs-csi-driver.exe gs://$CLOUDSDK_CORE_PROJECT-bucket/
+    ```
+## Preparing the cluster
+1. Go back to your k/k workspace where the patched kubernetes binaries were built and bring up a cluster using those binaries.
+    
+    **WARNING:** This wil tear down an existing e2e cluster in your project.
+    ```bash
+    PROJECT=${CLOUDSDK_CORE_PROJECT} \
+    NUM_NODES=2 \
+    NUM_WINDOWS_NODES=3 \
+    KUBE_GCE_ENABLE_IP_ALIASES=true \
+    KUBERNETES_NODE_PLATFORM=windows \
+    LOGGING_STACKDRIVER_RESOURCE_TYPES=new \
+    KUBERNETES_SKIP_CONFIRM=y \
+    go run ./hack/e2e.go -- --up
+    ```
+
+    **NOTE:** If the above command fails, try adding the flag `-get=false`.
+    ```bash
+    go run ./hack/e2e.go -get=false ...
+    ```
+
+    For further reference, [see](https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/windows/README-GCE-Windows-kube-up.md).
+
+2. Untaint the Windows nodes. The easiest way to do this is running the smoke test:
+
+    ```bash
+    ./cluster/gce/windows/smoke-test.sh
+    ```
+
+3. From Filestore driver workspace, set up cluster.
+    ```bash
+    PROJECT=${CLOUDSDK_CORE_PROJECT} ./deploy/kubernetes/cluster_setup.sh
+    ```
+
+4. On one of the Windows nodes, create an SMB share.
+    
+    Remember the name of this node, will be needed next step.
+    ```Powershell
+    $Password = ConvertTo-SecureString "<your password>" -AsPlainText -Force
+    New-LocalUser -Name smbuser -AccountNeverExpires -Password $Password
+    Add-LocalGroupMember -Group "Remote Desktop Users" -Member smbuser
+    New-Item -ItemType "directory" -Path "C:\SMBShare"
+    New-SmbShare -Name "SMBShare" -Path "C:\SMBShare" -FullAccess "smbuser"
+    ```
+
+5. Update necessary YAML files in Filestore driver workspace.
+    
+    Information you need:
+    - Project name
+    - Name of the node that hosts the SMB share
+    - Password from previous step
+    - Name of the nodes you will be deploying to.
+
+    List of files:
+    - `deploy/kubernetes/manifests/node_windows.yaml`
+    - `examples/kubernetes/windows/secrets.yaml`
+    - `examples/kubernetes/windows/pv.yaml`
+    - `examples/kubernetes/windows/nettest-pod.yaml`
+    - `examples/kubernetes/windows/powershell-nettest-pod.yaml`
+
+## Deploying
+[//]: # (Use kubelet commands instead of logging in)
+1. Log in to the nodes you specified in `examples/kubernetes/windows/nettest-pod.yaml` and `examples/kubernetes/windows/powershell-nettest-pod.yaml`.
+2. Run the filestore driver.
```

Comment:
Can you update your k/k CSI-prototype branch to fetch and run the filestore driver as a background process on each node at the end of the startup script?

The files you'd need to change would be the same as you did for https://github.com/kubernetes/kubernetes/pull/79447. I can send you an example for running a background process.

Reply:
Done.